lighthouse-core/test/report/html/renderer/report-ui-features-test.js

```diff
@@ -26,11 +26,23 @@ const TEMPLATE_FILE_REPORT = fs.readFileSync(__dirname +
   '/../../../../report/html/report-template.html', 'utf8');
 
 describe('ReportUIFeatures', () => {
-  let renderer;
-  let reportUIFeatures;
   let sampleResults;
   let dom;
 
+  /**
+   * @param {LH.JSON} lhr
+   */
+  function render(lhr) {
```

Comment:
render function looks 👍 

main issue is that it can be pretty slow, so we want to share rendered reports between tests as much as possible, but that's what you do here, so looks good to me